app/controllers/my-sessions/view.js

```diff
@@ -41,4 +41,4 @@ export default class extends Controller {
         this.set('isProposalDeleteModalOpen', false);
       });
   }
-}
\ No newline at end of file
+}
```

Comment:
Revert

Reply:
@maze-runnar

Reply:
@kushthedude i am removing new line but it is adding automatically , please help with this.

Reply:
@maze-runnar try using `Notepad` or any another `text-editor` to edit this file instead of your default one.